packages/component-library/src/HorizontalBarChart/HorizontalBarChart.js

```diff
@@ -269,6 +287,59 @@ const HorizontalBarChart = ({
                 />
               );
             })}
+          {grouped && (
+            <VictoryGroup
+              colorScale={theme.group.colorScale}
+              offset={barHeight * 1.2}
+            >
+              {groupedData.map(arr => {
+                return (
+                  <VictoryBar
+                    title="Horizontal Bar Chart"
+                    barWidth={barHeight}
+                    domainPadding={0}
+                    data={arr.map(d => ({
+                      dataKey: d[dataLabel],
+                      dataValue: d[dataValue],
+                      series: d[dataSeriesKey]
+                    }))}
+                    x="dataKey"
+                    y="dataValue"
+                    events={chartEvents(theme)}
+                    key={shortid.generate()}
+                    labels={arr.map(
+                      d => `${d[dataSeriesKey]}: ${d[dataValue]}`
+                    )}
+                    horizontal
+                    labelComponent={
+                      <VictoryTooltip
+                        x={325}
+                        y={0}
+                        orientation="bottom"
+                        pointerLength={0}
+                        cornerRadius={0}
+                        theme={theme}
+                      />
+                    }
+                    animate
+                  />
+                );
+              })}
+            </VictoryGroup>
+          )}
+          {grouped &&
```

Comment:
Again, can be wrapped in the condition above unless you're liking keeping this separate